### PR TITLE
add info on archiving community repos

### PR DESCRIPTION
**Pull request by kikisdeliveryservice**
Add info on moving sig repo into archives when a SIG is retired.
Closes: #3189 

cc: @castrojo

### CODE REVIEW

File: sig-governance.md [hunk 1]
```diff
@@ -103,6 +103,7 @@ The process for closing a SIG/WG is as follows:
   - Move all appropriate github repositories to an appropriate archive or a repo outside of the Kubernetes org
     - Each subproject a SIG owns must transfer ownership to a new SIG, outside the project, or be retired
     - File an issue with kubernetes/org if multiple repos that need to be retired
+    - Existing SIG repos in kubernetes/community will be moved into kubernetes/community/archive
```
Comment (nikhita) [inline sig-governance.md]: Perhaps `Move the existing SIG directory to the [archive](/archive) in kubernetes/community`?

Comment (castrojo) [reply sig-governance.md]: Yes it'd be better if we were explicit here.

Comment (kikisdeliveryservice) [reply sig-governance.md]: Thanks will do!

File: sig-governance.md [hunk 2]
```diff
@@ -103,6 +103,7 @@ The process for closing a SIG/WG is as follows:
   - Move all appropriate github repositories to an appropriate archive or a repo outside of the Kubernetes org
     - Each subproject a SIG owns must transfer ownership to a new SIG, outside the project, or be retired
     - File an issue with kubernetes/org if multiple repos that need to be retired
+    - Move the existing SIG directory into the [archive](/archive) in kubernetes/community
```
Comment (cblecker) [inline sig-governance.md]: This shouldn't be a subpoint of the github repo one. It falls under `Update `sigs.yaml` to reflect the removal of the SIG/WG`

Comment (kikisdeliveryservice) [reply sig-governance.md]: Done.

**Comment by castrojo** [comment]
/lgtm
/approve

Thanks for your help on this!

**Pull request merged**